src/app/home/models/room-creation-request.ts

```diff
@@ -0,0 +1,8 @@
+import { GameType } from "./game-type";
+
+export interface RoomCreationRequest {
```

Comment:
Maybe not request, since it will be the model of the room in firebase too? We are not calling an API, I think there might be no need for DTOs as such. 
Just a feeling though, I might be wrong.

Reply:
The difference but what I call a "request" and the model, is that one has `id`  and `players` maybe status `opened or closed` that represents the entity which is the model and one has none of these in my case it is the "request".

We are not defining the API but we are calling an API, firebase API that writes into the database, after all, we are not connecting directly to the database from the front end, nah?

I am open to changing the "request" to something else but I argue to separate between the "request" and the model.

Reply:
Hmm nope, you are right you smartass. 
It is also not necessary to offer me to still change it!